server/routes/visitJourney/selectVisitors.test.ts

```diff
@@ -127,13 +115,14 @@ testJourneys.forEach(journey => {
           offenderNo: 'A1234BC',
           dateOfBirth: '25 May 1988',
           location: 'location place',
+          activeAlerts: [],
+          restrictions: [],
         },
         visitRestriction: 'OPEN',
         visitReference: 'ab-cd-ef-gh',
       }
 
       prisonerVisitorsService.getVisitors.mockResolvedValue(returnData)
-      prisonerProfileService.getRestrictions.mockResolvedValue(restrictions)
```

Comment:
Can we remove `prisonerProfileService` entirely from this file now (it's included at the top...)?